src/lib/net/index.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * !TODO: add methods functionality, add extra functionality to the Networking (feel like something is missing), add middleware.
+ */
+
+import { DirectoryClass } from "./classes/directory";
+import { RemoteClass } from "./classes/remote";
+import { RouterClass } from "./classes/router";
+
+/* Types */
+import {
+	DirectoryDeclaration,
+	DirectoryObjectDeclaration,
+	EndpointsDeclaration,
+	RemoteObjectDeclaration,
+	RouterDeclaration,
+} from "./types";
+
+export namespace Network {
+	export namespace Method {
```

Comment:
The reason I've been pitching this with different names is that we're trying to emulate the way a webserver functions, so GET is the client asking the server for data, POST is the client telling the server about something. PATCH/UPDATE is the server telling the client something and it sounds like it.